src/classes/STG_PanelHealthCheck_CTRL.cls

```diff
@@ -290,6 +291,53 @@ public without sharing class STG_PanelHealthCheck_CTRL extends STG_Panel {
     }
 
     /*********************************************************************************************************
+    * @description Verifies HouseHold RecordType is not the default RecordType for the current User.
+    */
+    @TestVisible
+    private void verifyAccountDefaultRecordType() {
+        try {
+           
+            Id npspHouseholdRecordType = UTIL_CustomSettingsFacade.getOrgContactsSettings().npe01__HH_Account_RecordTypeID__c;
+            Id npspOneToOneRecordType = UTIL_CustomSettingsFacade.getOrgContactsSettings().npe01__One_to_One_RecordTypeID__c;
+            Map<Id,Schema.RecordTypeInfo> accountRecordTypeInfosById = Account.SObjectType.getDescribe().getRecordTypeInfosById();
+
+            //Check if the Household and the One To One RecordRecord Type in the Settings is valid. 
+            //This should never happen, unless the User manually updated the Account Model Record Type in the Custom Settings.
+            if(
+                (npspHouseholdRecordType != null && accountRecordTypeInfosById.keySet().contains(npspHouseholdRecordType) == false) ||
```

Comment:
our current coding convention, https://sites.google.com/a/salesforce.com/sfdo-products-team/artifacts-docs/technical-development-resources/cumulus-coding-conventions, has a space between if (...) and for (...), as does the majority (though not all) of our code base.